XML_to_SON/grammar_parser_test.py

```diff
@@ -0,0 +1,329 @@
+import xml.etree.ElementTree as ET
+import re
+import pprint
+import json
+import os.path
+
+conversion_dict = {
+    'string': 'String',
+    'nonNegativeInteger': 'Int',
+    'boolean': 'Boolean',
+    'double': 'double',
+    'positiveInteger': 'Int',
+    'float': 'Real',
+    'duration': 'Int',
+    'integer': 'Int',
+    'nonPositiveInteger': 'Int',
+    'negativeInteger': 'Int',
+    'long': 'Real',
+    'int': 'Int',
+    'token': 'String'
+}
+
+def get_element_type(node):
+    for child in node:
+        if child.tag == f"{ns}data" and 'type' in child.attrib:
+            xml_type = child.attrib['type']
+            json_type = conversion_dict.get(xml_type, 'Unknown')
+            return json_type
```

Comment:
Probably OK to combine there
```suggestion
            return conversion_dict.get(xml_type, 'Unknown')
```